lib/nerdtree/creator.vim

```diff
@@ -38,6 +38,29 @@ function! s:Creator.BufNamePrefix()
     return 'NERD_tree_'
 endfunction
 
+" FUNCTION: s:Creator.CreateExploreTree(dir) {{{1
+function! s:Creator.CreateExploreTree(dir)
+    try
+        let path = g:NERDTreePath.New(a:dir)
+    catch /^NERDTree.InvalidArgumentsError/
+        call nerdtree#echo('Invalid directory name:' . a:dir)
```

Comment:
Great work catching this, Initially I was going to suggest changing the `echo` to `echoError` but since we already use echo in the `CreateWindowTree`, It is best to leave it alone for the sake of consistency.